<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphical editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Figure to draw:</string>
    </property>
    <widget class="QRadioButton" name="rectButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rect</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ellipse</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>241</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Figure settings:</string>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>91</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Center:</string>
     </property>
     <widget class="QLabel" name="xLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="yLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="heightEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="widthEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
   </widget>
   <widget class="Frame" name="frame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>441</width>
      <height>361</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
   </widget>
   <widget class="QLabel" name="mouseLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouse coordinates: (0;0)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Frame</class>
   <extends>QFrame</extends>
   <header>frame.h</header>
   <container>1</container>
   <slots>
    <signal>sendCoordinates(QString)</signal>
    <signal>ready()</signal>
    <slot>getInfo(int,int)</slot>
    <slot>getRect(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>startDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frame</sender>
   <signal>sendCoordinates(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>getCoordinates(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>368</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>frame</sender>
   <signal>ready()</signal>
   <receiver>MainWindow</receiver>
   <slot>sendInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>sendItems(int,int)</signal>
   <receiver>frame</receiver>
   <slot>getInfo(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>394</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rectButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>rectSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>rectSet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>sendRect(bool)</signal>
   <receiver>frame</receiver>
   <slot>getRect(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sendItems(int,int)</signal>
  <signal>sendRect(bool)</signal>
  <slot>startDialog()</slot>
  <slot>getCoordinates(QString)</slot>
  <slot>sendInfo()</slot>
  <slot>rectSet()</slot>
 </slots>
</ui>
